src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/renametool/ConfirmationModalPage.java

```diff
@@ -0,0 +1,45 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.special.renametool;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialPageObject;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class ConfirmationModalPage extends SpecialPageObject {
+
+  //TODO: Move it to components
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)